app/Http/Controllers/Articles/ArticlesController.php

```diff
@@ -32,11 +32,11 @@ public function show(Article $article)
         ]);
     }
 
-    public function create()
+    public function create(Request $request)
     {
         $tags = Tag::all();
         $selectedTags = old('tags', []);
-        $series = Series::all();
+        $series = $request->user()->series;
         $selectedSeries = old('series');
```

Comment:
Let's place all of these in the array below since there's no use for the variables themselves.